contactsummary/tests/phpunit/CRM/Contactsummary/Utils/ContractSummaryTestTrait.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+/**
+ * A trait with helper methods to be reused among this extension's tests
+ */
+trait ContractSummaryTestTrait {
+
+  /**
+   * Creates absence period from specified start and end dates and return its ID
+   *
+   * @param $startDate
+   * @param $endDate
+   *
+   * @return int
+   */
+  protected function createAbsencePeriod($startDate, $endDate)  {
+    $params = array('name'=> $startDate,'start_date'=>$startDate, 'end_date'=>$endDate);
+    $period = civicrm_api3('HRAbsencePeriod', 'create', $params);
+    return $period['id'];
+  }
+
+  /**
+   * Creates absence entitlements for specific leave type
```

Comment:
I think the method documentation can be improved a little:
- It creates a single entitlement, so it should say: "Creates an absence entitlement"
- I don't think we need the "for specific leave type" part, unless the method required a $leaveType parameter. The way it is right now, it would be better something like "Creates an absence entitlement with the given $params"